config/wdio/wdio.conf.js

```diff
@@ -48,4 +48,9 @@ exports.config = {
     ui: 'bdd',
     timeout: 20000,
   },
+
+  before: () => {
+    /* global browser */
+    browser.setViewportSize({ width: 1050, height: 895, name: 'default' });
```

Comment:
Would we want to define this as a terra-size ?

Reply:
the Terra variable isn't available here for some reason.